detector/fcal/BeamCal_o1_v01_geo.cpp

```diff
@@ -45,8 +49,27 @@ static DD4hep::Geometry::Ref_t create_detector(DD4hep::Geometry::LCDD& lcdd,
   const double bcalInnerR = dimensions.inner_r();
   const double bcalOuterR = dimensions.outer_r();
   const double bcalInnerZ = dimensions.inner_z();
+  const std::string lcddname(lcdd.header().name());
   const double bcalThickness = DD4hep::Layering(xmlBeamCal).totalThickness();
   const double bcalCentreZ = bcalInnerZ+bcalThickness*0.5;
+  //  const double bcalOuterZ = dimensions.outer_z();
+  const double bcalOuterZ = bcalInnerZ+bcalThickness;
+  const std::string lcddcomment(lcdd.header().comment());
+
+  std::cout << "\nBeamCal Z_outer = " << bcalOuterZ/dd4hep::mm << " mm\n";
+  if ( lcddname.find("ILD") != std::string::npos && bcalOuterZ/dd4hep::mm > 3420 ) {
```

Comment:
Uhh, now. that is not what I meant.
I meant you should write a test, like the tests in https://github.com/iLCSoft/lcgeo/tree/master/lcgeoTests, and not hard code things like this into the driver.